react/src/components/organisms/Overlay/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './style.css';
+
+class Overlay extends Component {
+  state = {
+    showContent: false
+  };
+  toggleContent = () => {
+    this.setState((currentState) => {
+      return {
+        showContent: !currentState.showContent
+      };
+    }, () => {
+      // Set focus onto overlay-title after state is set.
+      if (this.state.showContent) {
+        document.getElementsByClassName('js-inline-overlay-title')[0].focus();
```

Comment:
Gatsby build shouldn't choke on this `document` ref since this is wrapped in a runtime method, but I'm wondering if this selector you're using is right, since it's always going to set focus to the first title?